about/policy/sgid/index.md

```diff
@@ -29,41 +46,52 @@ A corresponding record to the [Data Stewardship spreadsheet](https://docs.google
 - A link the the data page on the AGRC website
```

Comment:
```suggestion
- A link to the data page on the AGRC website
```

Reply:
I think I got that wrong. Should be "A link **on** the data page..." ?

Reply:
go ahead and edit your suggestion. I'll delete your comment.